p4rt_app/p4runtime/p4runtime_impl.cc

```diff
@@ -976,6 +982,41 @@ absl::Status P4RuntimeImpl::ConfigureAppDbTables(
       if (status.code() != google::rpc::OK) {
         return gutil::InvalidArgumentErrorBuilder() << status.message();
       }
+    } else if (table_type == table::Type::kExt) {
+      // Add table definition
+      /**
+       * For now send only Extension tables.In future when required, Fixed table
+       * definitions also can be inserted here
+       */
+      LOG(INFO) << "Add Table Definition for " << table_name;
+      sonic::InsertTableDefinition(tables_json, table);
+
+      publish_definition_set = true;
+    }
+  }
+
+  if (publish_definition_set) {
+     // Publish all tables at once and get one success/failure response for them
+    nlohmann::json info_json = nlohmann::json({});
+    info_json.push_back(
+        nlohmann::json::object_t::value_type("tables", tables_json));
```

Comment:
Can this be taken inside PublishExtTables...

Reply:
sure